docs/requirements.txt

```diff
@@ -1 +1,2 @@
 myst_parser
+docutils<0.17
```

Comment:
This fixes the issue with bullet points
- I tried deleting some white space, it didn't help
- using - instead of * also didn't help

-> I found some other issues which were solved by lowering the version of docutils, this is the latest version which works (<0.18 doesn't work). It is weird that it happened now, since there is no new release of ductils.

-> I suppose I will have to do it in all the previous versions of documentation in case we would rebuild it for some reason, is that right? @dschuermans

Reply:
I have no idea really 😅 

Could it be that at RTD they upgraded docutils recently? They're the ones performing the build eventually...

When I compared the changes between 2.2.0 branch and master branch, I couldn't really see any changes that would cause the bullet points to stop working.

But if that's the case, then we'd probably have to apply that change to the older version branches too, yes.